deploy/prometheus/prometheus.yml

```diff
@@ -7,4 +7,4 @@ scrape_configs:
       credentials: my-secret-metrics-token
     static_configs:
       - targets:
-        - localhost:1337
+        - app:8080
```

Comment:
I believe this won't work if `APP_PORT` is changed. I can suggest hardcoding `APP_PORT=8080` in the Dockerfile, as I don't think anybody would want to change it, since it's inside the container